cmd/oceantv/broadcast_machine.go

```diff
@@ -208,12 +206,7 @@ func (sm *broadcastStateMachine) handleTimeEvent(event timeEvent) {
 	case *vidforwardPermanentTransitionLiveToSlate:
 		withTimeout := sm.currentState.(stateWithTimeout)
 		if withTimeout.timedOut(event.Time) {
-			notifier.Send(
-				context.Background(),
-				sm.ctx.cfg.SKey,
-				"health",
-				"transition from live to slate timed out, staying in live state, check forwarding service",
-			)
+			sm.logAndNotify("transition from live to slate timed out, staying in live state, check forwarding service")
```

Comment:
Per feedback on Pull Request #179, I think logAndNotify should take a `notify.Kind` as its first argument.